integrations/clickup/client.py

```diff
@@ -0,0 +1,190 @@
+from typing import Any, AsyncGenerator, Dict, List
+
+from httpx import Timeout
+
+from loguru import logger
+
+
+from port_ocean.utils import http_async_client
+
+
+class ClickUpClient:
+    def __init__(self, clickup_token: str) -> None:
+        self.clickup_url = "https://api.clickup.com"
+        self.clickup_token = clickup_token
```

Comment:
Why do you have to add the token as a class variable? Why can't you use it directly in the auth headers?